frontend/src/pages/RegionalDashboard/index.js

```diff
@@ -83,11 +99,23 @@ export default function RegionalDashboard() {
 
   const filtersToApply = expandFilters(filters);
 
+  const showFilterWithMyRegions = () => {
+    // Exclude region filters we dont't have access to and show.
```

Comment:
same with this, could be moved to a regionHelpers file

Reply:
Agree, moved.